consistenthash/consistenthash.go

```diff
@@ -61,14 +61,51 @@ func (m *Map) Add(keys ...string) {
 	sort.Ints(m.keys)
 }
 
+// Gets the <amount> closest items in the hash to the provided key,
```

Comment:
First: Read https://golang.org/s/style#comment-sentences

Second: using <foo> for variables isn't idiomatic Go.